ballerina/tests/bytes_io.bal

```diff
@@ -182,67 +129,47 @@ function testFileWriteBytesFromStream() {
         byteContent[i] = s.toBytes();
         i += 1;
     }
-    var result = fileWriteBlocksFromStream(filePath, byteContent.toStream());
-    if (result is Error) {
-        test:assertFail(msg = result.message());
-    }
+    check fileWriteBlocksFromStream(filePath, byteContent.toStream());
 }
 
 @test:Config {dependsOn: [testFileWriteBytesFromStream]}
 function testFileReadBytesAsStream() returns Error? {
     string filePath = TEMP_DIR + "bytesFile3_B.txt";
-    var result = fileReadBlocksAsStream(filePath, 2);
+    stream<Block, Error?> result = check fileReadBlocksAsStream(filePath, 2);
     string expectedString = "Sheldon Cooper";
     byte[] byteArr = [];
-    if (result is stream<Block, Error?>) {
-        check result.forEach(function(Block val) {
-                                      foreach byte b in val {
-                                          byteArr.push(b);
-                                      }
-                                  });
-        string|error returnedString = langstring:fromBytes(byteArr);
-        if (returnedString is string) {
-            test:assertEquals(returnedString, expectedString);
-        } else {
-            test:assertFail(msg = returnedString.message());
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+
+    string|error returnedString = langstring:fromBytes(byteArr);
```

Comment:
Same as above

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4